data/localizations/categories/cs.yml

```diff
@@ -895,6 +895,8 @@ cs:
       name: Obuv
     aa-8-1:
       name: Atletická obuv
+    aa-8-10:
```

Comment:
the sort order 😢

Reply:
@tylerrowsell if you split the string by "-" then it can sort the resultant arrays so that [aa,8,2] will be before [aa,8,10]. I did so in the parser overriding the `<=>` method with that logic.

Feel free to poke me on slack if you want help or have questions

Reply:
oh gross. Didn't even notice this. I'll push a fix before merging.